lib/notification/adapter/ntfy.js

```diff
@@ -0,0 +1,50 @@
+import { markdown2Html } from '../../services/markdown.js';
+import { getJob } from '../../services/storage/jobStorage.js';
+import fetch from 'node-fetch';
+import utf8 from 'utf8';
+export const send = ({ serviceName, newListings, notificationConfig, jobKey }) => {
+  const { priority, server, topic } = notificationConfig.find((adapter) => adapter.id === 'ntfy').fields;
+  const job = getJob(jobKey);
+  const jobName = job == null ? jobKey : job.name;
+  const url = `${server}/${topic}`;
+  const promises = newListings.map((newListing) => {
+    const message = utf8.encode(
+      `Address: ${newListing.address} Size: ${newListing.size.replace(/2m/g, '$m^2$')} Price: ${newListing.price}`
+    );
+    return fetch(url, {
```

Comment:
while I totally appreciate not to use a lib for a simple http call, in this case, it might make sense to use one.

I'm thinking about using this: https://github.com/cityssm/node-ntfy-publish

If you decide against this, at least please don't use utf8 as an external dependency, coz it's unmaintained (last update 6 years ago and tons of issues)